_data/chats.yml

```diff
@@ -17,14 +14,8 @@ chats:
 - monero-pools:
   matrix: https://matrix.to/#/%23monero-pools:monero.social
 
-- monero-research-lab:
-  matrix: https://matrix.to/#/%23monero-research-lab:monero.social
-
-- monero-translations:
-  matrix: https://matrix.to/#/%23monero-translations:monero.social
-
-- monero-hardware:
-  matrix: https://matrix.to/#/%23monero-hardware:matrix.org
+- monero-support:
+    matrix: https://matrix.to/#/#monero-support:monero.social
 
-- monero-site:
```

Comment:
monero-site deleted?

Reply:
yes, deleted from the hangouts page but it's still on the workgroups page